common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -1,20 +1,52 @@
 import { css } from "@emotion/react";
-
-import { from } from "@guardian/source-foundations";
-import { neutral, space } from "@guardian/source-foundations";
+import { neutral, from, space, headline } from '@guardian/source-foundations';
+import { FirstPublished } from './FirstPublished';
 
 const LEFT_MARGIN_DESKTOP = 60;
 const SIDE_MARGIN = space[5];
 const SIDE_MARGIN_MOBILE = 10;
 
+const Header = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<header
+			css={css`
+				padding-right: ${space[3]}px;
+				display: flex;
+				flex-direction: column;
+			`}
+		>
+			{children}
+		</header>
+	);
+};
+
+const BlockTitle = ({ title }: { title: string }) => {
+	return (
+		<h2
+			css={css`
+				${headline.xxsmall({ fontWeight: 'bold' })}
+				margin-bottom: ${space[2]}px;
```

Comment:
I can't remember - what's the consensus for space vs remSpace in common-rendering?

Reply:
Yes, I tried switching it over and then things started changing in unpredictable ways in Chromatic so I switched it back to space.